src/overcooked_ai_py/data/graphics/objects.json

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
Is there a way to not have to have duplicated of the graphic data between `overcooked-ai` and `overcooked-demo`? What I'm imagining is that, if these are the same exact files as currently in `overcooked-demo`, `overcooked-demo` could read them in from here rather than having their own copy.